scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Job.java

```diff
@@ -120,11 +123,12 @@ public Job(int id) {
      * @param startTime start time
      * @param inErrorTime in error time
      * @param finishTime finish time
+     * @param description job description
      */
     public Job(int id, String name, String projectName, JobStatus status, JobPriority priority, String user,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 20 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)